pkg/dns/target.go

```diff
@@ -86,7 +85,8 @@ type ClusterGateway struct {
 }
 
 type ClusterAttributes struct {
-	Geo *GeoCode
+	CustomWeight *string
```

Comment:
I like that we are doing more of this with optional stuff it is a pattern you often see in other k8s controllers and APIs +1